packages/esm-outpatient-app/src/queue-patient-linelists/queue-linelist-filter.component.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  DatePicker,
+  DatePickerInput,
+  Form,
+  Grid,
+  Row,
+  RadioButtonGroup,
+  RadioButton,
+  Dropdown,
+  Column,
+  Button,
+  ButtonSet,
+  NumberInput,
+  ToggleSmall,
+} from 'carbon-components-react';
+import { filterType } from '../types/index';
+import { mockVisitTypes } from '../../__mocks__/visits.mock';
+import dayjs from 'dayjs';
+import { toDateObjectStrict, toOmrsIsoString, useLayoutType } from '@openmrs/esm-framework';
+import styles from './queue-linelist-filter.scss';
+
+interface QueueLinelistFilterProps {
+  toggleFilter: (filterMode: filterType) => void;
+  closePanel: () => void;
+}
+
+const QueueLinelistFilter: React.FC<QueueLinelistFilterProps> = ({ closePanel }) => {
+  const { t } = useTranslation();
+  const [gender, setGender] = useState('');
+  const [startAge, setStartAge] = useState(0);
```

Comment:
Wouldn't it be better not giving a default value and let the user themselves fill up the required age period.